packit_service/service/api/webhooks.py

```diff
@@ -88,6 +92,25 @@ def post(self):
 
         return "Webhook accepted. We thank you, Github.", HTTPStatus.ACCEPTED
 
+    @staticmethod
+    def interested():
+        """
+        Check whether we want to process this event.
+        :return: False if we are not interested in this kind of event
```

Comment:
google style comments :P

```suggestion
        Returns:
            `False` if we are not interested in this kind of event.
```